src/BW/Vkontakte.php

```diff
@@ -110,6 +115,9 @@ public function __construct(array $config = array())
         if (isset($config['state'])) {
             $this->setState($config['state']);
         }
+        if (isset($config['ipv6_disabled'])) {
+            $this->setIsIPv6Disabled((bool)$config['state']);
```

Comment:
Could you move this `bool` type cast into the setter? I think it'll be more appropriate there.